src/plugins/Network.js

```diff
@@ -470,6 +486,70 @@ module.exports = class Network extends OverlayPlugin {
                 option.selected = 'selected';
             }
         }
-        this.changeProfile();
+    }
+
+    /**
+     * Update mobile profile list UI according to the current active profile.
+     *
+     * @param {string} profile Profile name.
+     */
+    setActiveMobileProfile(profile) {
+        if (!profile) {
+            log.error('setActiveMobileProfile: Error : provided profile is empty');
+            return;
+        }
+        const mobileProfile = MOBILE_PROFILES.find((elem) => elem.name === profile);
+        if (!mobileProfile) {
+            return;
+        }
+
+        const options = this.select.getElementsByTagName('option');
+        for (let i = 0; i < options.length; i++) {
+            const option = options[i];
+            if (option.value === mobileProfile.name) {
+                option.selected = 'selected';
```

Comment:
you can break  here

Reply:
or maybe you should unselect every one else

Reply:
non breaking is OK